### ISSUE TITLE
OptiFine_1.14.4_HD_U_F4 won't install on my Windows 10

### ISSUE THREAD

**Issue by oHathor** [sp614x/optifine#3023]
When I double click the jar installer, the installer window opens, but closes after a second or so.
I then tried launching the installer from the command prompt:
```
C:\Users\helmut\AppData\Roaming\.minecraft\mods>java -jar OptiFine_1.14.4_HD_U_F4.jar
OptiFine Version: OptiFine_1.14.4_HD_U_F4
Minecraft Version: 1.14.4
OptiFine Edition: HD_U_F4
URL: file:/C:/Users/helmut/AppData/Roaming/.minecraft/mods/OptiFine_1.14.4_HD_U_F4.jar

C:\Users\helmut\AppData\Roaming\.minecraft\mods>java -version
java version "1.8.0_231"
Java(TM) SE Runtime Environment (build 1.8.0_231-b11)
Java HotSpot(TM) 64-Bit Server VM (build 25.231-b11, mixed mode)
```
No error message is displayed or written to file.
I removed all JRE from the system and reinstalled the version shown above (1.8.0_231) and verified that there is an appropriate entry for Java in the Windows Defender Firewall.
Anyhow, I suspect the issue is Windows related, but don't know what else I could do.
My System runs Windows 10 Pro, 1903

Do you have a hint for me?

**Comment by realjackasterisk** [comment]
I don't know why you're messing with your firewall of all things. Just use jarfix. https://johann.loefflmann.net/en/software/jarfix/index.html

**Comment by oHathor** [comment]
Messed file extension association is not the  problem. *.jar is correctly registered with _Java(TM) Platform SE binary_
As i wrote: the launcher is starting (showing the installer GUI for a second) but is immediately terminated.

**Comment by oHathor** [comment]
I run the installer again with the _-verbose_ option. The console output is in the file attached.

[optifine_installer_console_output.txt](https://github.com/sp614x/optifine/files/3778335/optifine_installer_console_output.txt)

**Comment by oHathor** [comment]
OK, it's a Java (and/or Windows) problem. The Java configurator shows the same behavior - is launched but terminates after a second.

**Comment by oHathor** [comment]
Replaced the 64-bit version of the JRE 1.8.0_231 with the 32-bit version.
This finally worked for me when executing the optifine installer from a command line (doubleclicking the jar still showed the same behavior as with the 64-bit version - which is crazy).

I assume the problem has been introduced by one of the latest Windows updates, since I successfully installed 1.14.4-OptiFine_HD_U_F3 on August 06.
